ta/concurrent/Android.mk

```diff
@@ -0,0 +1,7 @@
+include $(CLEAR_VARS)
```

Comment:
I checked some of the upstream android projects and it is apparently fine to do this.
See for example:
https://android.googlesource.com/platform/external/zxing/+/android-5.1.1_r37

Where the Android.mk includes Android.mk from the core/ subfolder.
I hope to avoid confusion between GNU and Android make files.